README.md

```diff
@@ -55,6 +55,8 @@ Files are assigned to object datastreams based on their basename, so a folder st
 
 would result in a two-page book.
 
+Each directory name will be converted into a number to be used as the sequence number of the page created from it.
```

Comment:
"Each" could be too broad...One might think that the books themselves would get a sequence number.

Also, not sure "converted" is appropriate?